deploy-docs/mkdocs-requirements.txt

```diff
@@ -11,3 +11,5 @@ mike
 plantuml-markdown
 pymdown-extensions
 python-markdown-math
+mkdocstrings
```

Comment:
How about making `mkdocs-requirements.txt` an input of `deploy-docs`?
https://github.com/takam5f2/autoware-github-actions/blob/9cc8a1bb37fdf5dfe7df1c63bd46b71fd98b4adb/deploy-docs/action.yaml#L23

Then, we don't have to send a PR to this repository every time.

Reply:
Thank you for kind help. I didn't notice that users are allow to configure their own `mkdocs-requirements.txt`.

 I will try to make `mkdocs-requirements.txt` and input it to `deploy-docs`.   
This PR is going to be closed at this moment.

Reply:
Ah, no. Currently, it's not supported.

Reply:
So I meant, how about extending this action in another approach?

Reply:
cc @KeisukeShima What do you think about this?

Reply:
Sorry for my misunderstanding. I have understood that updating `deploy-docs` is more practical way rather than updating `mkdocs-requirements.txt`.  
I agree with you.